classes/module/Module.php

```diff
@@ -1774,22 +1670,13 @@ final public static function isModuleTrusted($module_name)
             }
         }
 
-        if ($default_country_modules_list_content === null) {
```

Comment:
Why do you remove the check of "trust" of partner modules ? (real question, I dont see the link with the PR subject)

Reply:
Because the file is not anymore written. So this code become dead code.